submissions/ORybka/dom-js/main.js

```diff
@@ -0,0 +1,222 @@
+    mapPosition: ['scale(1.2)', '-120%', '-30%'],
+    dead: [1, 1, 0],
+  },
+  {
+    id: 5,
+    name: 'targaryen',
+    slogan: '"Fire and Blood"',
+    description:
+      "House Targaryen of Dragonstone is a noble family of Valyrian descent who once ruled the Seven Kingdoms of Westeros. It eventually became the first royal house of the Seven Kingdoms, as House Targaryen of King's Landing. The only family of dragonlords who survived the Doom of Valyria, the Targaryens left the Valyrian Freehold twelve years before the Doom.",
+    mapPosition: ['scale(1.3)', '-80%', '0'],
+    dead: [1, 1, 1],
+  },
+  {
+    id: 6,
+    name: 'tyrell',
+    slogan: '"Growing Strong"',
+    description:
+      'House Tyrell of Highgarden is an extinct Great House of Westeros. It ruled over the Reach, a vast, fertile, and heavily-populated region of southwestern Westeros, from their castle-seat of Highgarden as Lords Paramount of the Reach and Wardens of the South after taking control of the region from House Gardener during the Targaryen conquest.',
+    mapPosition: ['scale(1.2)', '-40%', '-45%'],
+    dead: [1, 1, 1],
+  },
+  {
+    id: 7,
+    name: 'baratheon',
+    slogan: '"Ours is the Fury"',
+    description:
+      "House Baratheon of Storm's End is a Great House of Westeros. A cadet branch was formerly the royal house, but House Lannister took control of the throne, though lost it to House Targaryen. House Baratheon traditionally rules the Stormlands on the eastern coast of Westeros, aptly named for its frequent storms, from their seat of Storm's End.",
+    mapPosition: ['scale(1.1)', '-40%', '-10%'],
+    dead: [1, 1, 1],
+  },
+  {
+    id: 8,
+    name: 'martell',
+    slogan: '"Unbowed, Unbent, Unbroken"',
+    description: "House Martell of Sunspear is one of the Great Houses of Westeros. It rules the peninsula of Dorne in the far south of the continent from their castle Sunspear. Though loyal to the Iron Throne, the Martells were never conquered by the Targaryens and pursued a more isolated role in wider political events since Robert's Rebellion.",
+    mapPosition: ['scale(1.1)', '20%', '-20%'],
+    dead: [1, 1, 1],
+  },
+];
+
+let button;
+let buttonLi;
+let image;
+let imageId;
+let imageCollection = [];
+let previousId = 0;
+const number = 3;
+const time = 1450;
+const houseElement = document.querySelector('.house');
+const shieldElement = document.querySelector('.shield');
+const sloganElement = document.querySelector('.slogan');
+const descriptionElement = document.querySelector('.description');
+const background = document.querySelector('.content-container');
+const menu = document.querySelector('.menu-container');
+const navbar = document.querySelector('.navbar');
+const images = document.querySelector('.content-img-container');
+const hiddenIcon = document.querySelector('.icon');
+const header = document.querySelector('.header-container');
+
+function createList() {
+  houses.forEach(({ id, name }) => {
```

Comment:
Please, use here [DocumentFragment](https://developer.mozilla.org/ru/docs/Web/API/DocumentFragment). It will help you to reduce DOM operations